openquake/hazardlib/geo/surface/gridded.py

```diff
@@ -127,27 +130,37 @@ def get_top_edge_depth(self):
         """
         return self.mesh.depths.min()
 
+     
     def get_strike(self):
         """
-        Compute surface's strike as decimal degrees in a range ``[0, 360)``.
-
-        The actual definition of the strike might depend on surface geometry.
-
+        p = point on the plane derived from plane_fit function.
+        the point is normalised and the angle between the 
+        normalised vector and the YZ plane is calculated.  
+        
```

Comment:
I do not understand this text. It should explain what the method does.

Reply:
It has been rephrased.